src/operators/multipliedoperator.jl

```diff
@@ -23,6 +23,8 @@ const TimedOperator{O} = MultipliedOperator{O,<:Function}
 const UntimedOperator{O} = MultipliedOperator{O,<:Union{Real,Nothing}}
```

Comment:
This honestly seems to be getting a bit out of hand, a `MultipliedOperator{O,<:Nothing}` should probably never exist, and just be a `O`, thus alleviating the need of the `const TimedUnion`, which is a also not a great name for timed + untimed operators, which are just any `MultipliedOperators`... If nothing else, you could create a trivially multiplied operator by using the hard `One()` from VectorInterface, which should in principle not actually execute multiplications?